### ISSUE TITLE
new release

### ISSUE THREAD

**Issue by matjazp** [Lukasa/hyper#364]
Can you please make a new release? 0.7.0 is more than a year old and there are many new commits and bug fixes in development branch since 9/2016.

**Comment by johejo** [comment]
I think so, too.
Are there any goals that must be achieved by the release of the next version?

**Comment by PrimozGodec** [comment]
I agree with @matjazp and @johejo. The new release would make our work much simpler.

**Comment by meeas** [comment]
Kindly adding to this request.  Requirements of "hyperframe<4.0,>=3.2" is now causing library conflicts with other tools in environments where virtenv can not be used.  The hyperframe package is at 5.1.0 and I have other tools requiring >=4.0.1
